tests/test-cases.js

```diff
@@ -585,6 +585,15 @@ var CORE_PARSER_TESTS = [
 			data: [['a', 'b', 'c'], ['']],
 			errors: []
 		}
+	},
+	{
+		description: "Duplicate header names",
```

Comment:
Could you add a test with also transformHeader function? As we are "manipulating" the headers I think we need to ensure that both functionalities work together.